packages/widgetbook/lib/src/fields/duration_field.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+
+import 'field.dart';
+import 'field_codec.dart';
+import 'field_type.dart';
+
+class DurationField extends Field<Duration> {
+  DurationField({
+    required super.name,
+    required Duration super.initialValue,
+    super.onChanged,
+  }) : super(
+          type: FieldType.doubleInput,
+          codec: durationCodec,
+        );
+
+  @override
+  Widget toWidget(BuildContext context, String label, Duration? currentValue) {
+    return TextFormField(
+      initialValue:
+          codec.toParam(currentValue ?? initialValue ?? Duration.zero),
```

Comment:
Consider extracting `Duration.zero` to a `static const` variable named `defaultDuration` _(same style as `ColorField`)_